crates/isograph_schema/src/magic_mutation_fields.rs

```diff
@@ -271,89 +287,12 @@ impl UnvalidatedSchema {
         d: &GraphQLDirective<ConstantValue>,
     ) -> ProcessTypeDefinitionResult<Option<MagicMutationFieldInfo>> {
         if d.name.item == *EXPOSE_FIELD_DIRECTIVE {
-            Ok(Some(self.validate_magic_mutation_directive(d)?))
+            Ok(from_graph_ql_directive(d).ok())
```

Comment:
`from_graph_ql_directive(d)?` — We should probably propagate the error here, i.e. we should not return `Ok(None)` if we fail to deserialize

Reply:
What a beginner error :(